scrapers/base.py

```diff
@@ -43,3 +44,24 @@ def get_html_content(self, url):
             print "Url is not reachable. Error logged on: {}".format(
                 strftime("%Y-%m-%d %H:%M:%S", gmtime()))
             log.info(str(err))
+
+    def local_store(self, data):
+        """Writes content from the website to file.
+        Usage::
+             the file name to write the data
+        :param_train_data: the filename
+        :rtype: outputs data to file.
+        """
+        file_name = "news"
+        if file_name:
+            with open(os.path.dirname(__file__) +"/data/"+"news"+".json", "r+") as output_file:
+                try:
+                    existing_data = json.load(output_file)
+                except:
+                    existing_data = {'news': []}
+                finally:
+                    if type(existing_data) is not dict:
+                        existing_data = {'news': []}
+                existing_data['news'] += data
```

Comment:
Does this mean that the functionality above here isn't applied to s3? Also, does it mean there is no limit to the file size that is being stored?